src/utility/ota/OTA-portenta-h7.cpp

```diff
@@ -26,13 +26,16 @@
 #include <Arduino_DebugUtils.h>
 #include <Arduino_Portenta_OTA.h>
 
+#include <WiFi.h>
+#include <Ethernet.h>
+
 #include "../watchdog/Watchdog.h"
 
 /******************************************************************************
  * FUNCTION DEFINITION
  ******************************************************************************/
 
-int portenta_h7_onOTARequest(char const * ota_url)
+int portenta_h7_onOTARequest(char const * ota_url, bool use_ethernet)
```

Comment:
```suggestion
int portenta_h7_onOTARequest(char const * ota_url, bool const use_ethernet)
```
I'd suggest using const-correctness as much as possible. As I see you are already doing it everywhere else I guess you just missed that one :wink:

Reply:
Fixed https://github.com/arduino-libraries/ArduinoIoTCloud/pull/328/commits/8d33a522dda9d7498b75b738d2013427502b2b05